data_model/src/query/mod.rs

```diff
@@ -1025,15 +1025,15 @@ pub mod trigger {
     }
 
     impl Query for FindTriggerById {
-        type Output = Trigger<TriggeringFilterBox, OptimizedExecutable>;
+        type Output = Trigger<TriggeringFilterBox>;
     }
 
     impl Query for FindTriggerKeyValueByIdAndKey {
```

Comment:
why do we have this query?

Reply:
I tracked it down to https://github.com/hyperledger/iroha/issues/1986, couldn't find any further motivation
I'd imagine it's what you would use if you receive `ExecuteTriggerEvent` and want to inspect the trigger for example